pkg/scheduler/api/node_info.go

```diff
@@ -185,6 +185,10 @@ func (ni *NodeInfo) setRevocableZone(node *v1.Node) {
 // Check node if enable Oversubscription and set Oversubscription resources
 // Only support oversubscription cpu and memory resource for this version
 func (ni *NodeInfo) setOversubscription(node *v1.Node) {
```

Comment:
BTW, if the function mainly focuses on check node, why not return a boolen?

Reply:
it is not care of the return